src/type/type.ts

```diff
@@ -0,0 +1,27 @@
+export interface SampleFolder {
+  folder: {
+    id: string;
+    name: string;
+    owner: {
+      id: number;
+      name: string;
+      profileImageSource: string;
+    };
+    links: [
+      {
+        id: number;
+        createdAt: string;
+        url: string;
+        title: string;
+        description: string;
+        imageSource: string;
+      }
+    ];
```

Comment:
P2:
link type을 외부에서 참조할 수 있게 따로 정의하시고 아래처럼 사용하시는 걸 추천드립니다.

```suggestion
    links: LinkProps[]
```